app/controllers/admins/settings_controller.rb

```diff
@@ -0,0 +1,27 @@
+require 'singleton'
+
+module Admins
+  class SettingsController < BaseController
+    include Singleton
```

Comment:
I haven't seen this module before! Do you have a link to documentation / a blog post about this?